Bottom: 7b1772f2a262fe361f13bc3c32a8748f8841eae8
Top:    2e11885c7c7b4260a2f82d876f9a990c762a6571
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-05-17 18:09:07 +0100

Refresh of conditional-compile-different

---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 460719e..f69b538 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -214,20 +214,26 @@ step_do_export (HID_Attr_Val * options)
         poly_DelContour (&curc);
       }
 
+#if 1
     silk_objects = object3d_from_silk_within_area (piece, TOP_SIDE);
     board_outline_list = g_list_concat (board_outline_list, silk_objects);
 
     silk_objects = object3d_from_silk_within_area (piece, BOTTOM_SIDE);
     board_outline_list = g_list_concat (board_outline_list, silk_objects);
+#endif
 
+#if 1
     mask_objects = object3d_from_soldermask_within_area (piece, TOP_SIDE);
     board_outline_list = g_list_concat (board_outline_list, mask_objects);
 
     mask_objects = object3d_from_soldermask_within_area (piece, BOTTOM_SIDE);
     board_outline_list = g_list_concat (board_outline_list, mask_objects);
+#endif
 
+#if 1
     copper_layer_objects = object3d_from_copper_layers_within_area (piece);
     board_outline_list = g_list_concat (board_outline_list, copper_layer_objects);
+#endif
 
   } while ((piece = piece->f) != board_outline);
   poly_Free (&board_outline);
